### PR TITLE
Added MinerGames

### PR DESCRIPTION
**Pull request by TheMCraft**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/minergames/manifest.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "bedwars": {
+      "name": "Bedwars",
+      "color": "#AA0000",
+      "url": "https://www.minergames.com/game/gungame",
+      "command": "/join bedwars"
+    }
+  },
+  "discord": {
+    "server_id": 816762698275618894,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/minergames/manifest.json:48]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress your images.

File: minecraft_servers/minergames/manifest.json (hunk 1 above)
Comment (TheMCraft) [reply minecraft_servers/minergames/manifest.json]: I have compressed the images and set it to false.

**Review by JakobBruening — APPROVED** [review]
Looks good, thank you! :)

**Pull request merged**